rust-toolchain.toml

```diff
@@ -2,5 +2,5 @@
 channel = "{{ toolchain }}"
 {%- if toolchain == "nightly" %}
 components = ["rust-src"]
-targets = ["riscv32imc-unknown-none-elf"]
+targets = ["{{ rust_target }}"]
```

Comment:
This might go wrong in the Xtensa case? I don't think rustup knows anything about Xtensa targets. I haven't tried it but we might need a riscv feature gate?

Reply:
For xtensa targets we will only have:
```toml
[toolchain]
channel = "esp"
```
The component and targets are inside an if statement that will be only true for RISC-V targets. The condition is being discussed in https://github.com/esp-rs/esp-template/pull/87#discussion_r1179995917

Reply:
Ah yes sorry, the diff made it look weird, my bad!